src/Taxonomies/Term.php

```diff
@@ -277,6 +278,14 @@ public function set($key, $value)
         return $this;
     }
 
+    private function getDirtyArray()
```

Comment:
Can this be made `public`, like it is in `Entry` and `User`?

Reply:
Yep, done!